src/usr/local/www/system_advanced_notifications.php

```diff
@@ -346,6 +382,43 @@
 
 $form->add($section);
 
+$section = new Form_Section('Telegram');
+
+$section->addInput(new Form_Checkbox(
+	'disable_telegram',
```

Comment:
We haven't formalized this yet but we are trying to avoid making any more "negative" options like this, where you enable a checkbox to disable a feature. The action and the wording of the option should match, ideally. You can still do this and default it to enabled by wording the option differently and changing the default behavior.

Additionally, we're trying to cut down on ambiguous options where it isn't clear from quickly glancing at `config.xml` whether or not the option is enabled. To work around that, explicitly set a value for the option either way, for example set it to `enabled` or `disabled`.

See this recent change for an example: https://github.com/pfsense/pfsense/commit/e5c4f2a7d977fb1fd6c7b4446e187486b72285be

Reply:
I've taken the approach of using set / unset. When you look at the XML file if the option "Enabled" is there under the "Telegram" hierarchy, then Telegram notifications are enable, if it's missing then disabled.